### PR TITLE
update to 3.28

### PR DESCRIPTION
**Pull request by miguelcobain**
A harmless PR that should be able to get those of us still stuck in 3.x world.
This should make updating to 3.28 possible, without deprecations, and consequently making the update to 4.x possible.

### CODE REVIEW

**Comment by miguelcobain** [comment]
@amkirwan can I help moving this PR forward?

File: addon/services/ember-oauth2.js [hunk 1]
```diff
@@ -107,7 +110,7 @@ export default Service.extend(Evented, {
    * @param {Function} callback Optional callback
    */
 
-  handleRedirect: on('redirect', function(hash, callback) {
+  handleRedirect: on('redirect', function (hash, callback) {
```
Comment (amkirwan) [inline addon/services/ember-oauth2.js:113]: nit: can you remove the spaces between function and the (). That seems like odd formatting to me. I see this in a few places in the changes

**Review by amkirwan — CHANGES_REQUESTED** [review]
Looks good overall. Haven't worked with Ember for awhile so I am assuming some of these changes work wit hthe latest version

File: addon/services/ember-oauth2.js (hunk 1 above)
Comment (miguelcobain) [reply addon/services/ember-oauth2.js:113]: >  I didn't do this myself. This is just the prettier config that comes out of the box with ember cli.

In a way, this is the default style of ember apps.

We can change prettier config, though.

**Comment by miguelcobain** [comment]
@amkirwan I replaced travis with GH actions. Tests are passing in Ember 3 and 5! (and one would assume ember 4 works too).

image
🔗  https://github.com/miguelcobain/ember-oauth2/actions/runs/7353253569

**Comment by miguelcobain** [comment]
These changes were never released

**Pull request merged**